### ISSUE TITLE
Support custom initialisers for -initWithViewClass:, -initWithLayerClass:

### ISSUE THREAD

**Issue by secretiverhyme** [facebook/AsyncDisplayKit#108]
See #106.  The usual approach—

```objective-c
[[ASDisplayNode alloc] initWithViewClass:[ASTableView class]]
```

—doesn't support initialiser-only configuration like UITableViewStyle.

**Comment by mtitolo** [comment]
+1 running into this constraint right now

**Comment by secretiverhyme** [comment]
Implemented in #235.
